test/linodes/linodes.bats

```diff
@@ -65,13 +66,33 @@ load '../common'
     assert_output --partial 'root_pass	root_pass is required'
 }
 
+@test "it should create a linode without an image and not boot" {
+    local linode_type=$(linode-cli linodes types --text --no-headers --format="id" | xargs | awk '{ print $1 }')
+    local linode_region=$(linode-cli regions list --format="id"  --text --no-headers | xargs | awk '{ print $1 }')
+    run linode-cli linodes create \
+        --no-defaults \
+        --label='cli-2' \
+        --type=$linode_type \
+        --region=$linode_region \
+        --root_pass $random_pass
+    local linode_id=$(linode-cli linodes list --format="id" --text --no-headers)
+
+    until [ $(linode-cli linodes view $linode_id --format="status" --text --no-headers) = "offline" ]; do
```

Comment:
This test would mean more if you check that the status wasn't still "provisioning"